deploy-manage/tools/snapshot-and-restore/s3-repository.md

```diff
@@ -377,6 +377,13 @@ AWS instances resolve S3 endpoints to a public IP. If the {{es}} instances resid
 
 Instances residing in a public subnet in an AWS VPC will connect to S3 via the VPC’s internet gateway and not be bandwidth limited by the VPC’s NAT instance.
 
+## Replicating objects [repository-s3-replicating-objects]
+
+AWS S3 supports [replication of objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html), both within a single region and across regions.
+Unfortunately this replication is not compatible with {{es}} snapshots.
```

Comment:
The original proposal is going to place the 5 lines in the same paragraph, i'm going to propose to divide the content in 3 different paragraphs.

```suggestion
AWS S3 supports [replication of objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication.html), both within a single region and across regions. However, this replication mechanism is not compatible with {{es}} snapshots.

```